templates/plugins/generic/htmlArticleGalley/display.tpl

```diff
@@ -32,26 +32,26 @@
 
 	{include file="frontend/components/footer.tpl"}
 
-{* Default style if CSS isn't attached to the HTML Galley *}
-{if $boolEmbeddedCss === false}
+	{* Default style if CSS isn't attached to the HTML Galley *}
```

Comment:
We can safely delete this method of injecting CSS to HTML galley, as well as associated variables, after switching to modified addStyle() method, like: https://github.com/pkp/immersion/blob/master/ImmersionThemePlugin.inc.php#L31

Reply:
Done and pushed